src/qibo_cloud_backends/braket_client.py

```diff
@@ -0,0 +1,65 @@
+import time
+
+from braket.aws import AwsDevice
+from braket.devices import LocalSimulator
+from qibo import Circuit as QiboCircuit
+from qibo.backends import NumpyBackend
+from qibo.config import raise_error
+from qibo.result import MeasurementOutcomes
+
+from qibo_cloud_backends.braket_translation import to_braket
+
+
+class BraketClientBackend(NumpyBackend):
+    def __init__(self, device=None, verbatim_circuit=False, verbosity=False):
+        """Backend for the remote execution of AWS circuits on the AWS backends.
+
+        Args:
+            device (str): The string representing the ARN of the Braket device. Defaults to Braket's statevector LocalSimulator,
+                          LocalSimulator("default"). Other devices are Braket's density matrix simulator,
+                          LocalSimulator("braket_dm"), or any other QPUs.
```

Comment:
I also notice that the docstring isn't very clear. I've modified it. 
Also, I've allowed the string `"braket_dm"` to be passed into `device` now.

Reply:
> Also, I've allowed the string `"braket_dm"` to be passed into `device` now.

Uhm, that's a good idea, but maybe you could define your own encoding for `LocalSimulator`, as there are multiple options (the only one missing now is the analog Hamiltonian simulation).

Something like `local_simulator:braket_dm`, and you split on the first `:`, if the first part is `local_simulator`, you pass the rest of the string to the `LocalSimulator` class.

Reply:
Good idea! That will be `LocalSimulator("braket_ahs")`. Can I suggest to leave analog Hamiltonian simulation for further development? We did not encounter using it in the development of `braket_client`. Hence I will prefer to leave analog Hamiltonian to a later time after I've gotten the hang of using it -- will open an issue and work from there!

Reply:
The solution is just to make it work with any local simulator, agnostic wrt the actual usage.

Just replace the current handling with:
```py
kind = device.split(":")[0]

self.device = AwsDevice(device) if kind != "local_simulator" else LocalSimulator(device.split(":")[1])
```
and that's it.

Reply:
I found this tutorial on how to run on the `braket_ahs`: https://docs.aws.amazon.com/braket/latest/developerguide/braket-get-started-hello-ahs.html 

Apparently we cannot simply call `device = LocalSimulator("braket_ahs")` and run circuits on it. There will be an implementation error:
`NotImplementedError: <class 'braket.analog_hamiltonian_simulator.rydberg.rydberg_simulator.RydbergAtomSimulator'> does not support qubit gate-based programs`. 

Let me work on it as part of an issue rather than integrate it now, for it will take time.

Reply:
@mho291 is not required to spend too much time on the AHS right now. If you want to investigate it, feel free of course. But I'd suggest you to split an issue, and do your investigation in a separate PR in any case. It's truly an additional feature (and in case it should also be advertised as such).